src/iris/api.py

```diff
@@ -1791,6 +1791,13 @@ def on_post(self, req, resp):
                                                    `current_step`, `active`, `application_id`, `bucket_id`)
                            VALUES (:plan_id, :created, :context, 0, :active, :application_id, :bucket_id)''',
                         data).lastrowid
```

Comment:
fix the flake8 errors and make sure the circleci tests are passing

Reply:
fixed